site/content/docs/cicd/silent.md

```diff
@@ -0,0 +1,11 @@
+# Suppressing prompts with —-silent
```

Comment:
This content is a duplicate of what is in cicd.md, can we please delete this file?